frontend/src/lib/api/apiClient.ts

```diff
@@ -1,4 +1,4 @@
-import { ApiError, isApiErrorResponse } from './apiError';
+import { ApiError, isApiErrorResponse } from './ApiError';
```

Comment:
this is an old problem, didnyou sync with dev? this change should not be required